numpy/lib/npyio.py

```diff
@@ -1687,6 +1708,8 @@ def genfromtxt(fname, dtype=float, comments='#', delimiter=None,
         to None the system default is used. The default value is 'bytes'.
 
         .. versionadded:: 1.14.0
+    ndmin : int, optional
+        Same parameter as `loadtxt`
```

Comment:
This will require a `versionadded` note.

Reply:
which version should I put?

Reply:
I think it is `1.23`

Reply:
I'd put it at the bottom of the `ndim` description. The `max_rows` docs also have a `versionadded` note, I'd use that as example if I were you.